webapi/CopilotChat/Skills/ChatSkills/SemanticChatMemoryExtractor.cs

```diff
@@ -51,7 +53,7 @@ internal static class SemanticChatMemoryExtractor
                 );
                 foreach (var item in semanticMemory.Items)
                 {
-                    await CreateMemoryAsync(item, chatId, kernel.Memory, memoryName, options);
+                    await CreateMemoryAsync(item, chatId, kernel.Memory, memoryName, options, logger, context.CancellationToken);
```

Comment:
The cancellation token in SKContext is an obsolete member now.  Add a CancellationToken param to your method signature instead of retrieving it from SKContext.

Reply:
added